src/stratis_cli/_actions/_constants.py

```diff
@@ -24,3 +26,7 @@
 FILESYSTEM_INTERFACE = "org.storage.stratis2.filesystem"
 POOL_INTERFACE = "org.storage.stratis2.pool.r1"
 BLOCKDEV_INTERFACE = "org.storage.stratis2.blockdev"
+
+MAXIMUM_STRATISD_VERSION = "2.1.0"
+MINIMUM_STRATISD_VERSION = "2.0.0"
+assert Version(MINIMUM_STRATISD_VERSION) < Version(MAXIMUM_STRATISD_VERSION)
```

Comment:
Which min and max versions should be accepted currently? 
Pre-release versions cannot be specified here due to the format of the spec string https://github.com/stratis-storage/stratis-cli/pull/552/files#diff-c9e6aa089520ef1da260e2c79d1d9e8fR36. This can be revised.

Reply:
> Which min and max versions should be accepted currently?
> Pre-release versions cannot be specified here due to the format of the spec string https://github.com/stratis-storage/stratis-cli/pull/552/files#diff-c9e6aa089520ef1da260e2c79d1d9e8fR36. This can be revised.

Let's talk about this in standup.